src/software/amazon/ion/IonWriter.java

```diff
@@ -14,14 +14,15 @@
 
 package software.amazon.ion;
 
+import software.amazon.ion.system.IonTextWriterBuilder;
```

Comment:
I separated the code style changes and fiddled a bit with my IDE settings to avoid this in the future. 

Do we have a style guide I can use as reference?